rootfs/Dockerfile

```diff
@@ -14,6 +14,25 @@
 
 ARG BASE_IMAGE
 
+# Prepare Lua files and their dependencies.
+FROM ${BASE_IMAGE} as lua_builder
+RUN apk add \
+  make \
+  wget \
+  git
+RUN curl -sSL -o /tmp/luarocks.tar.gz https://github.com/luarocks/luarocks/archive/refs/tags/v3.9.2.tar.gz \
+  && echo "a0b36cd68586cd79966d0106bb2e5a4f5523327867995fd66bee4237062b3e3b */tmp/luarocks.tar.gz" | sha256sum -c - \
+  && tar -C /tmp -xzf /tmp/luarocks.tar.gz \
+  && cd /tmp/luarocks* \
+  && ./configure \
+  && make \
+  && make install
+COPY etc/nginx/lua /lua
+WORKDIR /lua
+RUN luarocks make --local --tree /luarocks ingress-nginx-lua-main-1.rockspec
```

Comment:
In the future, will we need to add a different luarocks file for each release version?

Reply:
no, we would just keep changing the same rockspec file without versioning it. because we don't ship the lua modules of ingress-nginx separately and they are relevant only within ingress-nginx.

Reply:
got it